qutebrowser/config/configcommands.py

```diff
@@ -250,6 +250,72 @@ def config_unset(self, option, temp=False):
         with self._handle_config_error():
             self._config.unset(option, save_yaml=not temp)
 
+    @cmdutils.register(instance='config-commands')
+    @cmdutils.argument('option', completion=configmodel.structure_option)
+    def config_add(
```

Comment:
The more I think about it, the more I think this should be split into `:config-add-dict` and `:config-add-list` or so. The signatures of the two things are quite different (`:config-add-dict option key value` vs. `:config-add-list option value`). 

@milogert what do you think about that?

Reply:
Yeah, that's fair. The dict+list method seemed like (as you said below) that it was doing too much. Splitting them seems like a good way to go.